File: Web-Api/Model/Entities/Pick.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
namespace Pickfc.Model.Entities
{
    public class Pick
    {
        [Key]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int ID { get; set; }
        public int RoundID { get; set; }
        public int GameID { get; set; }
        public int PlayerID { get; set; }
        public int TeamID { get; set; }
        public int RoundNumber { get; set; }
        public string Result { get; set; } = string.Empty;
        public DateTime Time { get; set; }
        public DateTime Timestamped { get; set; }
    }
}

File: Client-App/app/_model/pick.ts
import { Team } from "./team"

export interface Pick{
  id: number
  roundID: number
  gameID: number
  playerID: number
  teamID: number
  roundNumber: number
  playerName: string
  playerPic: string
  result: string
  time: Date
  timestamped: Date
  team: Team
}
